### PR TITLE
(hotfix)[impuls] add operator waypoint

### PR DESCRIPTION
**Pull request by PetervDooren**
Proper fix to this should be done in the action server. But in the meantime we need a working robot

### CODE REVIEW

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]
Actually there shouldn't be any operator predefined in the WM

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]
No need to give it a pose yet, I think

**Comment by PetervDooren** [comment]
Yes, it shouldnt be predefined but fixing the bug in the action server will take time. Therefore this hotfix, to be removed later once the bug is fixed. 
The pose does need to be predefined. Currently it is not updated by the action server

**Comment by MatthijsBurgh** [comment]
@PetervDooren I will test without a pose this weekend

**Comment by PetervDooren** [comment]
@MatthijsBurgh tested last Friday and I can say that at the very least the predefined pose is not overwritten.

**Comment by MatthijsBurgh** [comment]
Pose is needed
https://github.com/tue-robotics/ed/blob/3ce42e4898ae3677b043d9448a47210d0b676124/src/ed.cpp#L375-L376

File: models/impuls/model.sdf [hunk 1]
```diff
@@ -96,7 +96,8 @@
       <name>hallway_table</name>
       <uri>model://ddw/table</uri>
     </include>
-    <model name="operator">
+    <model name="operator" type="waypoint"> <!-- hotfix! operator should not need to be predefined. Fix in action server -->
+      <pose>2.0 1.0 0 0 0 0</pose>
```
Comment (MatthijsBurgh) [inline models/impuls/model.sdf:100]:
```suggestion
      <pose>0 0 0 0 0 0</pose>
```

**Review by MatthijsBurgh — APPROVED** [review]
Just as a very dirty hotfix.

**Comment by PetervDooren** [comment]
I fear that setting it at the origin will fail all the time. I set it next to the operator table so a demo will actually succeed

**Comment by MatthijsBurgh** [comment]
> I fear that setting it at the origin will fail all the time. I set it next to the operator table so a demo will actually succeed

To my knowledge, it is set correctly at the time it is being used. But it needs to be in the WM, of the correct type and have a pose.
If the hack in the action_server is so bad that it needs to be at the correct spot. I don't even approve this PR.

**Review by MatthijsBurgh — APPROVED** [review]

**Pull request merged**